.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject
```

Comment:
Please delete this file. It has no use for end user because normally they should not update code in vendor folder.